### ISSUE TITLE
when the first line is a blank line or there are macro compilations, it fails to remove correctly.

### ISSUE THREAD

**Issue by FirmianaMarsili** [JeremyCaron/vscode-csharp-organize-usings#21]
in my example, I only used `UnityEngine`.  <br>

this is my setting:
```    
    "[csharp]": {
        "editor.codeActionsOnSave": [
            "source.organizeImports"
        ]
    },
    "csharpOrganizeUsings.sortOrder": "System",
    "editor.formatOnSave": true,
```

Is it possible to have an optionto skip using statements wrapped in macros?<br>
and this is my screenshot.<br>

![organize_sings](https://github.com/user-attachments/assets/391c63b2-5760-4644-ae42-ca833147cac7)
![organize_sings_blank](https://github.com/user-attachments/assets/26e0d593-2d48-4c4a-81c3-d42ff57d6615)

**Comment by JeremyCaron** [comment]
@FirmianaMarsili I think I've stopped the bleeding here on preporcessor directives (and fixed this other issue you mentioned in the second gif).  I've released the fix as a prerelease version (1.1.0).  Could you please give it a try and let me know what you think?  I'd like at least one other person to try it before I do a real release.  Thanks!

**Comment by FirmianaMarsili** [comment]
Yes, that's exactly the issue you mentioned. If it's a standard .NET project, like a console output project, then there is no problem. However, some of my projects rely on `Unity` for automatic management, and for some other projects, I manually control the build process. In such cases, the plugin doesn't work, and it keeps showing that message.

**Comment by SpiffyGH** [comment]
I have encountered this in version 1.0.6

https://github.com/user-attachments/assets/68a9371a-2830-4832-8dc1-47acaa477ca1

**Comment by JeremyCaron** [comment]
Released the fix I did for this today.  @SpiffyGH if v1.2.0 doesn't solve what you were seeing please open a new issue.  Thanks!
